pyne/r2s.py

```diff
@@ -222,3 +223,18 @@ def total_photon_source_intensity(m, tag_name, sub_voxel=False):
             sv_data = ve_data[num_e_groups*svid:num_e_groups*(svid+1)]
             intensity += vol * np.sum(sv_data)
     return intensity
+
+def e_bins_to_photon_source_h5m(filename, e_bins):
+    """
+    This function is used to add 'energy_boundaries' attribute for PyNE R2S
+    photon source file 'source_x.h5m'.
+    Parameters
+    ----------
+    filename : string
+        Filename of the 'source_x.h5m'.
+    e_bins : numpy array
+        Energy boundaries, unit: MeV.
+    """
+    with tb.open_file(filename, 'r+') as h5f:
+        h5f.create_array('/', 'energy_boundaries', obj=e_bins)
```

Comment:
should we put this deeper in the HDF hierarchy?  (perhaps other info should also be deeper??)